runner/runners/invoke.go

```diff
@@ -61,6 +61,7 @@ func (inv *Invoker) Run(cmd *runner.Command, id runner.RunID) (abortCh chan<- st
 func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struct{}, updateCh chan runner.RunStatus) (r runner.RunStatus) {
 	log.Infof("run. id: %v", id)
 	defer func() {
+		inv.stat.Latency(stats.WorkerTaskLatency_ms).Time().Stop()
```

Comment:
I mean, I see other examples of timers where we call Time() first, and Stop() later - if we are immediately initializing the timer and calling stop in the defer func, aren't all of these measurements basically zero?